src/main/java/com/google/devtools/build/lib/bazel/repository/JarDecompressor.java

```diff
@@ -147,6 +148,7 @@ protected String createBuildFileWithSrcjar(String baseName, String srcjarBaseNam
             "    srcs = [",
             "        '" + baseName + "',",
             "        '" + srcjarBaseName + "',",
```

Comment:
I think you need to remove this line.

Reply:
Yes, of course...

Reply:
Does any test fail if you *don't* remove this line?

Reply:
Yes, i forgot to remove that line. If i keep the srcjarBaseName line the testTargetIsSource test fails:

bazel test //src/test/java/com/google/devtools/build/lib/rules/repository:RepositoryTests

There was 1 failure:
1) testTargetIsSource(com.google.devtools.build.lib.rules.repository.JarDecompressorTest)
java.lang.AssertionError: Not true that <"# DO NOT EDIT: automatically generated BUILD.bazel file for maven_jar rule tester
java_import(
    name = 'jar',
    jars = ['foo-sources.jar'],
    srcjar = 'foo-sources.jar',
    visibility = ['//visibility:public']
)

filegroup(
    name = 'file',
    srcs = [
        'foo-sources.jar',
        'foo-sources.jar',
    ],
    visibility = ['//visibility:public']
)
"> contains <"srcs = [
        'foo-sources.jar',
    ],">
    at com.google.devtools.build.lib.rules.repository.JarDecompressorTest.testTargetIsSource(JarDecompressorTest.java:106)